packages/config/src/log/messages.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const { WILDCARD_ALL: FUNCTIONS_CONFIG_WILDCARD_ALL } = require('../functions_config')
+
+const { log } = require('./logger')
+
+const warnLegacyFunctionsDirectory = ({ config = {}, logs }) => {
```

Comment:
```suggestion
const warnLegacyFunctionsDirectory = ({ config, logs }) => {
```

`config` should always be defined here.